src/linux/linux_io.c

```diff
@@ -665,6 +765,7 @@ int run_io(struct eventloop *loop, const struct cmdline_config *config)
 	} else {
 		ret = run_io_all_interfaces(loop, config, handler, ARRAY_SIZE(handler));
 	}
+	unlink(UDS_PATH);
```

Comment:
I would put the unlink to the end of run_io_only_local() and run_io_all_interfaces().

Reply:
done